service-front/content/guidance/People_Certificate providers.md

```diff
@@ -1,5 +1,10 @@
 # Certificate providers
 
+<div class="moj-banner moj-banner__message">
+    <h class="heading-small"><a href="https://www.gov.uk/guidance/making-and-registering-an-lpa-during-the-coronavirus-outbreak#the-certificate-provider-and-donor-conversation">Read our guidance</a>  on being a certificate provider during the corona virus outbreak</h>
```

Comment:
This (and the others) should probably be in `<p>...</p>` tags, since it isn't really a heading for content. Also, `<h>` only exists in the dream world where XHTML2 got approved instead of HTML5, it would have been great...

Reply:
Wait I might be confused now I've scrolled down and have also spotted these are `.md`

Reply:
I think we render it out via controllers. Not 100%

Quite right on the _h_ and _p_ things though

Reply:
I agree, have changed from <h> to <p>.

I had to insert HTML code in the markdown content to allow the wanted styling, which otherwise was tricky.